mj_envs/hand_manipulation_suite/hammer_v0.py

```diff
@@ -92,7 +92,7 @@ def gs(self):
         qpos = self.data.qpos.ravel().copy()
         qvel = self.data.qvel.ravel().copy()
         board_pos = self.model.body_pos[self.model.body_name2id('nail_board')].copy()
-        target_pos = self.data.site_xpos[self.target_obj_sid].ravel().copy()
+        target_pos = self.model.site_pos[self.target_obj_sid].ravel().copy()
```

Comment:
model.body_pos provides the position wrt the body frame.
data.body_xpos provides the positions wrt the world frame.

In information needed here are wrt the world frame. I, therefore, believe that the original code is correct. Thoughts?